dashboard/static/css/forms.css

```diff
@@ -28,5 +28,4 @@ textarea, input {
     cursor: default;
     font-size: .7em;
     margin-top: 2px;
-    text-align: center;
```

Comment:
The question mark bubble looks like this when the `text-align: center` rule is removed:

![Question mark bubble with the question mark mis-aligned](https://cloud.githubusercontent.com/assets/933396/13617181/5820e8b8-e54b-11e5-8d29-fbbef3c9aa0e.png)